src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantsController.kt

```diff
@@ -67,14 +74,32 @@ class ParticipantsController(
     return SimpleSuccessResponsePayload()
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping
+  @Operation(summary = "List participants and their assigned projects.")
+  fun listParticipant(): ListParticipantResponsePayload {
```

Comment:
Nit: This name should probably be `listParticipants` and the payload class name should be `ListParticipantsResponsePayload`.